dcos/dcos-ui
# DCOS-53347: add error badges to Jobs form

## Testing
Go to the jobs form, make some errors happen, confirm that the tabs get badges

For example:
1. Create a job with the id "test-id"
2. Select the "Command Only" radio input
3. Submit
4. The "General" tab should be badged with 1 error

## Trade-offs
The messages in the ErrorsAlert banner are treated differently than in the Create Service form:
- They aren't de-duplicated
- The error path is prepended to the error messages

Not de-duplicating the errors causes the error badge count not to match the number of list items in the ErrorsAlert banner

## Dependencies
[dcos-ui-plugins-private PR](https://github.com/mesosphere/dcos-ui-plugins-private/pull/1055)

## Screenshots
<img width="1632" alt="Screen Shot 2019-08-07 at 5 14 50 PM" src="https://user-images.githubusercontent.com/2313998/62659541-a724ec00-b939-11e9-8bd7-9f75233465f7.png">
